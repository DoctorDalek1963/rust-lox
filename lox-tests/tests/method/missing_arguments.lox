class Foo {
  method(a, b) {
    print a;
    print b;
  }
}

Foo().method(1); // expect runtime error: Expected 2 arguments but got 1.
