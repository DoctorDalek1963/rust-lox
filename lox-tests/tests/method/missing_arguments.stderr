ERROR: Expected 2 arguments in function call but got 1
 --> 8:1
  |
8 | Foo().method(1); // expect runtime error: Expected 2 arguments but got 1.
  | ^-------------^
